examples/Mediawiki/Contribs.yaml

```diff
@@ -0,0 +1,91 @@
+suite: Contribs
+description: List user contributions
+type: RPC
+
+use-fixtures:
+  - Alice
+
+# All parts of the request/response pair (aka interaction step) to be reused.
+# We'll probably see this one a lot. Can we safely/sanely share prototypes
+# between test suites somehow? 
```

Comment:
This would be nice! I've written the edit one too many times. Maybe a file of global prototypes?

Reply:
Yes, but what variables can they use? They don't act like functions, they don't have parameters, they just exist in the context of the suite... It's unclear to me how to make that work nicely.